pkgs/build-support/cc-wrapper/setup-hook.sh

```diff
@@ -69,10 +69,12 @@ ccWrapper_addCVars () {
     getHostRoleEnvHook
 
     if [ -d "$1/include" ]; then
+        (! echo $NIX_CFLAGS_COMPILE | grep $1 > /dev/null) &&
```

Comment:
Please add quotes around variables that aren't meant to be word split.

Reply:
I know lots of things will break if store paths have space in them, but I think for correctness the `$1` passed to grep should also be quoted.

Reply:
make sense!